src/MusicStore/config.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
+    "ApplicationInsights": {
+        "InstrumentationKey": "797d1481-9988-48f2-b41b-f4e305cd9207"
+    },
     "DefaultAdminUsername": "Administrator@test.com",
     "DefaultAdminPassword": "YouShouldChangeThisPassword1!",
```

Comment:
We should not encourage people to put passwords in config.json. We should discuss how to handle the user secrets correctly.